sway-lib-std/src/tx.sw

```diff
@@ -6,6 +6,7 @@ use ::address::Address;
 use ::context::registers::instrs_start;
 use ::contract_id::ContractId;
 use ::intrinsics::is_reference_type;
+use ::option::Option;
```

Comment:
Maybe move this under `::mem` to keep in alphabetical order?

Reply:
can't wait for forc-fmt to handle this!

Reply:
https://github.com/FuelLabs/sway/issues/2226